config-example.yml

```diff
@@ -101,13 +102,16 @@ graphhopper:
   #### Encoded Values ####
 
   # Add additional information to every edge. Used for path details (#1548) and custom models (docs/core/custom-models.md)
-  # Possible values: road_class,road_class_link,road_environment,max_speed,road_access
-  #   surface,smoothness,max_width,max_height,max_weight,max_weight_except,hgv,max_axle_load,max_length,
-  #   hazmat,hazmat_tunnel,hazmat_water,lanes,osm_way_id,toll,track_type,mtb_rating,hike_rating,horse_rating,
-  #   country,curvature,average_slope,max_slope,car_temporal_access,bike_temporal_access,foot_temporal_access
-  # Private roads are blocked by default to disable this you can specify (also applies to other access encoded values like bike_access):
-  #   car_access|block_private=false
-  graph.encoded_values: car_access, car_average_speed, road_access
+  # Example values:
+  #   average_slope,country,curvature,hazmat,hgv,hike_rating,lanes,max_height,max_length, ...
+  # For more details and a full list see the documentation: https://github.com/graphhopper/graphhopper/blob/master/docs/core/custom-models.md
+  # Private roads are blocked by default. To disable this you can specify block_private=false as option:
+  #   car_access|block_private=false, bike_access|block_private=false
+  # By default we add the necessary encoded values for car.json, bike.json and foot.json (remove them if you do not need them)
+  graph.encoded_values: |
+    car_access, car_average_speed, country, road_class, roundabout
```

Comment:
It obviously is not detected by the yml parser as error, but for consistency please add a comma at the end of the string after `roundabout` in this line